tests/spread/core24-suites/plugins/colcon-msg-package/task.yaml

```diff
@@ -0,0 +1,44 @@
+summary: >-
+  Make sure ROS downloaded packages are properly sourced
+
+kill-timeout: 180m
+
+environment:
+  SNAP/colcon_msg_package: colcon-msg-package
+
+systems:
+  - ubuntu-22.04
```

Comment:
these are using 22.04, is that on purpose?

Reply:
no, but I will drop it from here and make the suite use 22.04

Reply:
fwiw, they could be handled by the global suite systems filter, which I switched to 22.04, but these tests will not work on non amd64 or arm64, so the filter is more specific